src/components/editor/editor-pane/editor-pane.tsx

```diff
@@ -157,16 +203,30 @@ export const EditorPane: React.FC<EditorPaneProps & ScrollProps> = ({ onContentC
   }), [t, editorPreferences])
 
   return (
-    <div className={'d-flex flex-column h-100'} onMouseEnter={onMakeScrollSource}>
+    <div className={'d-flex flex-column h-100 position-relative'} onMouseEnter={onMakeScrollSource}>
       <MaxLengthWarningModal show={showMaxLengthWarning} onHide={() => setShowMaxLengthWarning(false)} maxLength={maxLength}/>
       <ToolBar
         editor={editor}
       />
+      <ShowIf condition={showDropOverlay}>
+        <DropOverlay
+          onDrop={event => {
+            if (editor) {
+              onDrop(editor, event)
+              setShowDropOverlay(false)
+            }
+          }}
+          onDragLeave={() => setShowDropOverlay(false)}
```

Comment:
The editor changes a lot, so please use callbacks here.

Reply:
done